spring-cloud-stream/src/main/java/org/springframework/cloud/stream/annotation/EnableModule.java

```diff
@@ -47,7 +46,7 @@
 @Configuration
 @MessageEndpoint
 @Import({RedisServiceConfiguration.class, RabbitServiceConfiguration.class,
-	ChannelBindingAdapterConfiguration.class, CodecConfiguration.class, ChannelBindingAdapterRunner.class,
```

Comment:
I created XD-3380 to track this, giving it a try as it is the basis for some test support I want to provide for module authors (XD-3381)